test/common/run-tests

```diff
@@ -65,14 +65,25 @@ def print_test(test, print_tap=True):
         print("not ok " + test_name(test))
     sys.stdout.flush()
 
-def finish_test(opts, test):
+def finish_test(opts, test, affected_tests):
     """Returns if a test should retry or not
 
     Call test-policy on the test's output, print if needed.
 
     Return (retry_reason, exit_code). retry_reason can be None or a string.
     """
+
+    affected = len(affected_tests) < 3 and test.command[0].encode("utf-8") in affected_tests
+
     if test.process.returncode in [0, 77]:
```

Comment:
What is code `77`? Do we want to try again is such case as well?

Reply:
77 means "skip", and we don't want to retry these.

Reply:
Right, fixed so we only retry on success and not on skip.